library/utils.c

```diff
@@ -172,10 +194,10 @@ void ziti_logger(int level, const char *file, unsigned int line, const char *fun
     va_start(argp, fmt);
     char location[128];
     if (func && func[0]) {
-        snprintf(location, sizeof(location), "%s:%d %s()", file, line, func);
+        snprintf(location, sizeof(location), "%s:%d %s()", file + SOURCE_PATH_SIZE, line, func);
```

Comment:
I liked having the `ZITI_LOG_PREFIX` in the message (so you'd see stuff like `[2020-12-01T15:19:32.744Z] INFO ziti-tunnel-sdk-c:../lib/ziti_tunnel.c:44 ziti_tunneler_init(): Ziti Tunneler SDK (v0.7.17)`)

Reply:
it seemed redundant to me, we get full relative path of the files now so it is more clear where the file is coming from